centreon/www/front_src/src/Configuration/ConfigurationBase/api/useGetAll.ts

```diff
@@ -36,7 +38,15 @@ const useGetAll = ({
           sort
         }
       }),
-    getQueryKey: () => ['listResources', sortField, sortOrder, limit, page],
+    getQueryKey: () => [
+      'listResources',
+      sortField,
+      sortOrder,
+      limit,
+      page,
+      configuration?.resourceType,
+      filtersAtomKey
+    ],
```

Comment:
I think it's better to pass on key about the type of items (acc, token, etc...)
```suggestion
    getQueryKey: () => [
      'listResources',
      resourceKey,
      sortField,
      sortOrder,
      limit,
      page,
      configuration?.resourceType
    ],
```